runbooks/how-to-update-for-rda-api-changes.md

```diff
@@ -0,0 +1,53 @@
+      - [FissClaimStreamCallerIT](https://github.com/CMSgov/beneficiary-fhir-data/blob/master/apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/source/FissClaimStreamCallerIT.java)
+      - [FissClaimTransformerTest](https://github.com/CMSgov/beneficiary-fhir-data/blob/master/apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/source/FissClaimTransformerTest.java)
+      - [StandardGrpcRdaSourceIT](https://github.com/CMSgov/beneficiary-fhir-data/blob/master/apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/source/StandardGrpcRdaSourceIT.java)
+      - etc.
+4. Create FHIR mappings for new RDA fields
+   1. Pull new fields from RDA data dictionary
+   2. Reach out to FHIR SME to research new fields and identify appropriate FHIR mappings
+   3. Vet new mappings for what has value to customers, not all fields may get mapped
+   4. Create / update Claim / ClaimResponse model structure [in confluence](https://confluence.cms.gov/display/BCDA/PACA+FHIR+Resources)
+      - New mapping entries in tables
+      - Update Claim / ClaimResponse sample JSON responses for FISS / MCS claim types.
+5. Update [bfd-pipeline-rda-bridge](https://github.com/CMSgov/beneficiary-fhir-data/tree/master/apps/bfd-pipeline/bfd-pipeline-rda-bridge) module source code
+   1. Identify feasible synthetic CCW data values for corresponding new RDA fields
+   2. Update transformer with new field mappings
+   3. Update tests with new expectations
+      - May need to update the test RIF (csv) files with more/better data for new data fields
+6. Deploy SBX update
+   1. Create new synthetic RDA data using the synthetic CCW data from synthea
+      - Synthea release sets are in s3://bfd-public-test-data/data-synthetic/Synthea-Data/ (organized by release date)
+      - Directions to run the bridge are in the module [README.md](https://github.com/CMSgov/beneficiary-fhir-data/tree/master/apps/bfd-pipeline/bfd-pipeline-rda-bridge)
+      - Synthetic RDA messages for PROD-SBX get gzipped and copied to s3://bfd-prod-sbx-etl-577373831711/rda_api_messages/
+      - The manifest file gets updated with the new release sets
+7. Update FHIR API with new FHIR mappings
+   1. Add any new coding required coding systems
+      - Try to use existing systems where possible and where it enforces consistency
+      - PAC specific systems for FISS/MCS go in [BBCodingSystems](https://github.com/CMSgov/beneficiary-fhir-data/blob/master/apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/BBCodingSystems.java)
+   2. Update IT [endpoint responses](https://github.com/CMSgov/beneficiary-fhir-data/tree/master/apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2) to reflect new expectations
```

Comment:
Maybe add a step for `Update Blue Button static site` with these bullets.

- Clone the [BB static site repo](https://github.com/CMSgov/bcda-static-site)
- Open the [RDA API Data dictionary](https://docs.google.com/spreadsheets/d/19CGXLO9SGTgaSvACRVuuOatvs4z1NXHzxtI-zt_j8Qc/edit#gid=283461576) and export/download tab `B) Data Dictionary` to a CSV file.
- Replace the file `_data/rda_api_data_dictionary.csv` in BB static site repo with the exported CSV file.
- Test the static site locally to verify new variables can be loaded in browser.
- Check in change and push.
- Create PR in BB static site repo and request approval.

Reply:
Nice thanks for this, should this be before or after the FHIR API update?

Reply:
As long as its live in BB static site before the FHIR changes go live it's probably fine.  Maybe do both in the same sprint?